### PR TITLE
DevTools 110: Recorder and perf panels

### PR DESCRIPTION
**Pull request by sofiayem**
Documented:
- https://developer.chrome.com/blog/new-in-devtools-110/#perf
- https://developer.chrome.com/blog/new-in-devtools-110/#recorder-code
- https://developer.chrome.com/blog/new-in-devtools-110/#recorder-edit

The [selector types](https://developer.chrome.com/blog/new-in-devtools-110/#recorder-selector) were [already documented](http://localhost:8080/docs/devtools/recorder/reference/#selector), then I forgot to roll back the updates, so now no update is required. (๏_๏ )

Publish tomorrow.

### CODE REVIEW

**Review by rachelandrew — APPROVED** [review]

File: site/en/docs/devtools/recorder/reference/index.md [hunk 1]
```diff
@@ -154,6 +154,22 @@ By default, the **Recorder** replays the user flow as fast as it can. To underst
...
 {% endAside %}
 
+### Inspect code {: #inspect-code }
+
+To inspect the code of a user flow in various formats:
+
+1. Open a recording in the **Recorder** panel.
+1. Click **Show code** in the top right corner of the steps list.
+   {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/KhynxMityVxQeMJuNdU9.png", alt="The Show code button.", width="800", height="669" %}
+1. The **Recorder** shows a side-by-side view of the steps and their code. As you hover over a step, the **Recorder** highlights its respective code.
```
Comment (jecfish) [inline site/en/docs/devtools/recorder/reference/index.md:164]: Move to another step, show a video to illustrate the point if possible. 

"As you hover over a step, the **Recorder** highlights its respective code."

Comment (jecfish) [inline site/en/docs/devtools/recorder/reference/index.md:164]: Explicitly cal out that extension support this

File: site/en/docs/devtools/recorder/reference/index.md [hunk 2]
```diff
@@ -154,6 +154,22 @@ By default, the **Recorder** replays the user flow as fast as it can. To underst
...
+   {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/3K7idCdrqrZHmMR5Kczc.png", alt="The side-by-side view of the steps and their code.", width="800", height="685" %}
+1. Expand the format drop-down list to select a format that you use to [export user flows](#export-flows).
+
+   {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/fefVto1d2oXvkmBD9DPQ.png", alt="The format drop-down list.", width="800", height="685" %}
+
+   It can be one of the three default formats (JSON, [@puppeteer/replay](https://github.com/puppeteer/replay), [Puppeteer script](https://pptr.dev/)) or a format provided by an [extension](#recorder-extension).
+1. Proceed to debug your recording by [editing step parameters and values](#edit-steps). The code view isn't editable.
```
Comment (jecfish) [inline site/en/docs/devtools/recorder/reference/index.md:171]: The code will refresh according, but not editable.

**Review by jecfish — APPROVED** [review]
LGTM % comments

**Pull request merged**